django_recaptcha/fields.py

```diff
@@ -72,16 +92,17 @@ def validate(self, value):
                 remoteip=self.get_remote_ip(),
             )
 
-        except HTTPError:  # Catch timeouts, etc
-            raise ValidationError(
+        except HTTPError as e:  # Catch timeouts, etc
+            raise CaptchaHTTPError(
                 self.error_messages["captcha_error"], code="captcha_error"
-            )
+            ) from e
```

Comment:
Today I learned about exception chaining! 🚀 

https://docs.python.org/3/tutorial/errors.html#exception-chaining

Reply:
...I came from Java - I like exceptions and logging :)

Reply:
@alanverresen just saw this. Yeah, let me split this up and then let's talk about the logging stuff on the new PR. If you can hold comments until then, that'll keep things cleaner over on this PR.    ...I defiantly want to hear more about your thoughts on logging!

Reply:
I'd appreciate discussing logging in a new issue!

I can definitely see why people are not particularly interested in bots failing the captcha, that shouldn't have to be logged as a warning. Captcha failures due to the reCAPTCHA API being unavailable are definitely more concerning and should be monitored more closely.